tfjs-backend-webgl/src/backend_webgl.ts

```diff
@@ -176,6 +176,39 @@ export class MathBackendWebGL extends KernelBackend {
     return this.texData.numDataIds() - this.pendingDeletes;
   }
 
+  // Writes a new entry to the data store with a WebGL texture, and registers it
+  // to the texture manager.
+  writeTexture(
+      texture: WebGLTexture, shape: number[], dtype: DataType,
+      texHeight: number, texWidth: number, color: string): DataId {
```

Comment:
nit: `color` -> `channels` to keep consistent?

Reply:
Done.